compiler/ilgen/OMRVirtualMachineOperandStack.cpp

```diff
@@ -109,8 +109,9 @@ OMR::VirtualMachineOperandStack::Reload(TR::IlBuilder* b)
    }
 
 void
-OMR::VirtualMachineOperandStack::MergeInto(TR::VirtualMachineOperandStack* other, TR::IlBuilder* b)
+OMR::VirtualMachineOperandStack::MergeInto(TR::VirtualMachineState* o, TR::IlBuilder* b)
    {
+   TR::VirtualMachineOperandStack *other = (TR::VirtualMachineOperandStack *)o;
```

Comment:
Can you use a static cast here:

```suggestion
   TR::VirtualMachineOperandStack *other = static_cast<TR::VirtualMachineOperandStack *>(o);
```

I know VirtualMachineOperandArray uses a C-style cast but these can be error-prone so I would prefer to use C++ style casts in new/modified code.

Reply:
I have updated it.  I actually do not believe it is any more or less error-prone.  I didn't initially use the C++ static case because so much (almost all) of the code outside of tril uses C-style casts and I feel lightly spreading the static cast throughout just makes the code less consistent. The GC team experimented with the C++ static casts instead of C-style casts and quickly decided it was not worth it to switch everything as it did not catch any issues that C-style casts did not catch.